src/fragments/lib/datastore/native_common/advanced-workflows.mdx

```diff
@@ -0,0 +1,210 @@
+## Custom Primary Keys
+
+By default, all DataStore models have an `id` field by default that is automatically populated on the client with a UUID v4, allowing DataStore to generate non-colliding globally unique identifiers in a scalable way.
```

Comment:
```suggestion
By default, DataStore models have an `id` field that is automatically populated (non-colliding globally unique identifiers (UUID v4)) on the client by DataStore in a scalable way.
```

Reply:
Thanks for reviewing this PR, @kevinold! IMO, the original is a bit more readable, but could use some changes. What do you think about the following?
```suggestion
By default, DataStore models have an `id` field that is automatically populated on the client with a UUID v4, allowing DataStore to generate non-colliding globally unique identifiers in a scalable way.
```

Reply:
@david-mcafee Yes, this looks good to me.

Reply:
Thanks @kevinold !